README.md

```diff
@@ -1,6 +1,12 @@
 # teal.logger
 
 <!-- start badges -->
+
+[![CRAN Version](https://www.r-pkg.org/badges/version/teal.logger?color=green)](https://cran.r-project.org/package=teal.logger)
```

Comment:
Are we sure that this is the correct URL or should we wait adding the CRAN stats until we successfully release to CRAN?

Reply:
Yes @donyunardi this is standard url and I just provided the package name. This doesn't matter if the package is released or not. You can see this looks totally fine on this branch, and will get changed the moment the package gets published on CRAN

![image](https://github.com/insightsengineering/teal.logger/assets/133694481/e4ffb094-e67b-4752-846a-7add29dee82e)

Reply:
👌🏼